pkg/apis/serving/v1alpha1/service_defaults.go

```diff
@@ -40,9 +40,12 @@ func (s *Service) SetDefaults(ctx context.Context) {
 		if apis.IsInUpdate(ctx) {
 			old := apis.GetBaseline(ctx).(*Service)
 			if equality.Semantic.DeepEqual(old.Spec, s.Spec) {
-				return
+				s.SetAnnotations(old.GetAnnotations())
```

Comment:
I don't think we want to prohibit changes to all of the annotations.

Reply:
I have reverted those changes